### DISCUSSION TITLE
How to set VCPKG_CXX_FLAGS_DEBUG in Visual Studio? (using vcpkg manifest)

### DISCUSSION THREAD

**Discussion by sigmareaver** [microsoft/vcpkg#23076] [Q&A] [answered]
I have Visual Studio projects using the `vcpkg.json` manifest, and my projects' debug configuration defines:
`/D_SECURE_SCL=0 /D_HAS_ITERATOR_DEBUGGING=0`

I need my vcpkg dependencies to also compile with this, but I did not see any options for this in VS.

**Comment by autoantwort** [comment, marked as answer]
You have to create a new triplet for that and then use that new triplet. See https://github.com/microsoft/vcpkg/blob/master/docs/users/triplets.md#vcpkg_cxx_flags
